custom_dc/data_cloud.md

```diff
@@ -124,14 +124,27 @@ Now that everything is configured, and you have uploaded your data in Google Clo
 
 Every time you upload new input CSV or JSON files to Google Cloud Storage, you will need to rerun the job.
 
-To run the job:
+To run the job using the Cloud Console:
 
 1. Go to [https://console.cloud.google.com/run/jobs](https://console.cloud.google.com/run/jobs){: target="_blank"} for your project.
 1. From the list of jobs, click the link of the "datacommons-data" job you created above.
 1. Click **Execute**. It will take several minutes for the job to run. You can click the **Logs** tab to view the progress. 
 
 When it completes, to verify that the data has been loaded correctly, see the next step.
 
+#### Start the data management container in schema update mode {#schema-update-mode}
```

Comment:
Should this be "Run the data management job" for consistency with the previous header?

Reply:
yup, good catch, thanks!